ddclient.in

```diff
@@ -6428,6 +6438,185 @@ sub nic_hetzner_update {
     }
 }
 
+######################################################################
+## nic_inwx_examples
+######################################################################
+sub nic_inwx_examples {
+    return <<"EoEXAMPLE";
+o 'inwx'
+
+The 'inwx' protocol is an adaption of the 'dyndns2' protocol. This is nescessary
+because INWX uses a custom Update-URL-Format for IPv6-Support:
+
+'https://dyndns.inwx.com/nic/update?myip=<ipaddr>&myipv6=<ip6addr>'
+
+Please note (as of June 2024):
+- INWX does not support updating an IPv6 without including the IPv4 in the
+URL, unless the DynDNS-Account has been turned into IPv6-only manually by their support.
+- Including the IPv6 in the URL without 'myip' or with an empty value for '<ipaddr>'
+will result in the IPv4 (A-Record) beeing reset to '127.0.0.1'.
+- Updating the IPv4 only will delete the IPv6 (AAAA-Record).
+- 'dyndns.inwx.com' is not reachable via IPv6 (there is no AAAA-Record).
+- GET 'https://dyndns.inwx.com/nic/update' without further parameters will set the IPv4
+to the public IP of the requesting host.
```

Comment:
Does this also delete the IPv6 AAAA record?

Reply:
Yes, it does exactly the same as `https://dyndns.inwx.com/nic/update?myip=<ipaddr>`.